src/herbpy/action/pushing.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+import logging, numpy
+from prpy.action import ActionMethod
+from prpy.planning.base import PlanningError
+logger = logging.getLogger('herbpy')
+
+@ActionMethod
+def PushToPoseOnTable(robot, obj, table, goal_position, goal_radius, 
+                      manip=None, max_plan_duration=30.0, 
+                      shortcut_time=3., **kw_args):
+    """
+    @param robot The robot performing the push
+    @param obj The object to push
+    @param table The table kinbody the object is resting on
+    @param goal_position The desired (x,y) position of the object in world coordinates
+    @param goal_radius The max distance from goal_position for the object to be to 
+      still consider the goal achieved
+    @param manip The manipulator to use for the push - if None the active manipulator is used
+    @param max_plan_duration The max time to run the planner
+    @param shortcut_time The amount of time to spend shortcutting, if 0. no shortcutting is performed
+    """
+    # Get a push planner
+    from or_pushing.push_planner import PushPlanner
```

Comment:
Can you catch the `ImportError` here and throw an exception with a better error message? For example, "Unable to create PushPlanner. Is the `randomized_rearrangement_planning` repository checked out in your workspace?".